packit/api.py

```diff
@@ -1970,6 +1985,7 @@ def create_rpms(
                 upstream_ref=upstream_ref,
                 release_suffix=release_suffix,
                 merged_ref=merged_ref,
+                env=self.common_env(),
```

Comment:
I hope you don't get lost in the docs… I got lost last time I touched this area…

Reply:
Sorry, I don't follow, what docs?

Reply:
packit.dev

Reply:
If you mean which environment variables are available for which action, I just added it to the list here: https://github.com/packit/packit.dev/pull/975/files#diff-94d2421b8c9482d621ca048035692b1615f2cc21565f200376783afae080b0cfR264